cranelift/codegen/src/isa/mod.rs

```diff
@@ -120,57 +111,24 @@ macro_rules! isa_builder {
     }};
 }
 
-/// The "variant" for a given target. On one platform (x86-64), we have two
-/// backends, the "old" and "new" one; the new one is the default if included
-/// in the build configuration and not otherwise specified.
-#[derive(Clone, Copy, Debug)]
-pub enum BackendVariant {
-    /// Any backend available.
-    Any,
-    /// A "legacy" backend: one that operates using legalizations and encodings.
-    Legacy,
-    /// A backend built on `MachInst`s and the `VCode` framework.
-    MachInst,
-}
-
-impl Default for BackendVariant {
-    fn default() -> Self {
-        BackendVariant::Any
-    }
-}
-
 /// Look for an ISA for the given `triple`, selecting the backend variant given
 /// by `variant` if available.
-pub fn lookup_variant(triple: Triple, variant: BackendVariant) -> Result<Builder, LookupError> {
-    match (triple.architecture, variant) {
-        (Architecture::Riscv32 { .. }, _) | (Architecture::Riscv64 { .. }, _) => {
-            isa_builder!(riscv, (feature = "riscv"), triple)
-        }
-        (Architecture::X86_64, BackendVariant::Legacy) => {
-            isa_builder!(x86, (feature = "x86"), triple)
-        }
-        (Architecture::X86_64, BackendVariant::MachInst) => {
-            isa_builder!(x64, (feature = "x86"), triple)
-        }
-        #[cfg(not(feature = "old-x86-backend"))]
-        (Architecture::X86_64, BackendVariant::Any) => {
+pub fn lookup_variant(triple: Triple) -> Result<Builder, LookupError> {
```

Comment:
Can you rename this to just `lookup` (and remove the wrapper below)? This was added to accept the extra `variant` arg and so I don't think we need it any more.

Reply:
Done

Reply:
It looks like the latest force-push lost this update?

Reply:
Right, forgot to commit it again before force pushing.